src/Maths/math.vector.ts

```diff
@@ -5831,10 +5832,11 @@ export class Matrix {
         let b = 2.0 * n / height;
         let c = (f + n) / (f - n);
         let d = -2.0 * f * n / (f - n);
+        let rot = Math.tan(projectionPlaneTilt);

```

Comment:
What if `projectionPlaneTilt` = Math.PI/2 + n*Math.PI?

Reply:
When we approch Math.PI/2, things get really distorted so it shouldn't really be used, this is meant for small angles close to 0. 

But in case it would carry `NaN` somewhere, I'll put a safeguard